content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,100 @@ it does the following when a Pod is created:
+Starting with Kubernetes version 1.29, the legacy ServiceAccount token cleaner
+is enabled by default. It runs as part of the `kube-controller-manager` and
+checks every 24 hours to see if any legacy service account token has not been
+used in a *specified amount of time*. If so, the cleaner marks those tokens as
+invalid.
+
+The cleaner works by first checking the ConfigMap created by the
+`LegacyServiceAccountTokenTracking` feature. If the current time is a
+*specified amount of time* after the date in the ConfigMap, the cleaner then
+loops through the list of Secrets in the cluster and evaluates each Secret that
+has the type `kubernetes.io/service-account-token`.
+
+If a Secret meets all of the following conditions, the cleaner marks it as
+invalid:
+
+- The Secret is auto-generated, meaning that it is bi-directionally referenced
+  by a ServiceAccount.
+- The Secret is not currently mounted by any pods.
+- The Secret has not been used in a *specified amount of time* since it was
+  created or since it was last used.
+
+The cleaner marks an invalid Secret by adding a label called
+`kubernetes.io/legacy-token-invalid-since` to the Secret, with the current date
+as the value. If an invalid Secret is not used in a *specified amount of time*,
+the cleaner will delete it.
+
+{{< note >}}
+All the *specified amount of time* above is default to be one year. Users can
+configure this value through the `--legacy-service-account-token-clean-up-period`
+flag.
+{{< /note >}}
+
+If users use the invalidated auto-generated legacy token, the legacy token
+validator will
+- add an audit annotation for the key-value pair
+  `authentication.k8s.io/legacy-token-invalidated`:`secretName/Namepace`,
+- increment the `invalid_legacy_auto_token_uses_total` metric count,
+- update the Secret label `kubernetes.io/legacy-token-last-used` with the new
+  date,
+- return an error indicating that the legacy token has been invalidated.
+
+If received this validation error, users can update the Secret to remove the
+`kubernetes.io/legacy-token-invalid-since` label to temporarily allow use of
+this token.
+
+Here's an example of an auto-generated legacy token that has been marked with the
+`kubernetes.io/legacy-token-last-used` and `kubernetes.io/legacy-token-invalid-since`
+labels:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: build-robot-secret
+  namespace: default
+  labels:
+    kubernetes.io/legacy-token-last-used: 2022-10-24
+    kubernetes.io/legacy-token-invalid-since: 2023-10-25
+  annotations:
+    kubernetes.io/service-account.name: build-robot
+type: kubernetes.io/service-account-token
+```
+
+{{< note >}}
+The difference between the manually created legacy token and
+the auto-generated legacy token is that only auto-generated legacy token is
+bi-directionally referenced by the ServiceAccount, like:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: build-robot
+  namespace: default
+secrets:
+  - name: build-robot-secret
+```
+
```

Comment:
refer to the section "Create additional API tokens"

Reply:
Moved the note to the section "Create additional API tokens".